src/files/BrsFile.ts

```diff
@@ -334,94 +331,71 @@ export class BrsFile {
-                        let stmt = line.substring(idx).trim();
-                        stmt = stmt.replace(search.text, '');
-                        stmt = stmt.trim();
-
-                        let commentFlag: CommentFlag;
-
-                        //statement to disable EVERYTHING
-                        if (stmt.length === 0) {
-                            commentFlag = {
-                                file: this,
-                                //null means all codes
-                                codes: null,
-                                range: Range.create(lineIndex, idx, lineIndex, idx + search.text.length),
-                                affectedRange: affectedRange
-                            };
-
-                            //disable specific rules on the next line
-                        } else if (stmt.startsWith(':')) {
-                            stmt = stmt.replace(':', '');
-                            let codes = [] as number[];
-                            //starting position + search.text length + 1 for the colon
-                            let offset = idx + search.text.length + 1;
-                            let codeTokens = util.tokenizeByWhitespace(stmt);
-                            for (let codeToken of codeTokens) {
-                                let codeInt = parseInt(codeToken.text);
-                                //add a warning for unknown codes
-                                if (!diagnosticCodes.includes(codeInt)) {
-                                    this.diagnostics.push({
-                                        ...DiagnosticMessages.unknownDiagnosticCode(codeInt),
-                                        file: this,
-                                        range: Range.create(lineIndex, offset + codeToken.startIndex, lineIndex, offset + codeToken.startIndex + codeToken.text.length)
-                                    });
-                                } else {
-                                    codes.push(codeInt);
-                                }
-                            }
-                            if (codes.length > 0) {
-                                commentFlag = {
-                                    file: this,
-                                    codes: codes,
-                                    range: Range.create(lineIndex, idx, lineIndex, line.length),
-                                    affectedRange: affectedRange
-                                };
-                            }
-                        }
+            let affectedRange: Range;
+            if (tokenized.disableType === 'line') {
+                affectedRange = Range.create(token.range.start.line, 0, token.range.start.line, token.range.start.character);
+            } else if (tokenized.disableType === 'next-line') {
+                affectedRange = Range.create(token.range.start.line + 1, 0, token.range.start.line + 1, Number.MAX_SAFE_INTEGER);
```

Comment:
We used to find the length of the next line, but there's no real benefit to doing that, so just use MAX_SAFE_INTEGER instead because that will definitely match anything on the next line.